.circleci/config.yml

```diff
@@ -25,56 +32,71 @@ jobs:
             chmod +x linux-install-1.10.3.998.sh
             sudo ./linux-install-1.10.3.998.sh
       - run:
-          name: Install babashka
+          name: Install latest babashka
           command: |
             bash <(curl -s https://raw.githubusercontent.com/borkdude/babashka/master/install) --dir ~
             sudo mv ~/bb /usr/local/bin/bb
       - run:
           name: Run JVM tests
           command: |
             bb jvm-test
+      - run:
+          name: Run babashka tests
+          command: |
+            bb babashka-test
       - save_cache:
           paths:
             - ~/.m2
           key: v1-dependencies-{{ checksum "deps.edn" }}
-      - run:
-          name: Create uberjar
-          command: |
-            mkdir -p /tmp/release
-            lein do clean, uberjar
-            VERSION=$(cat resources/DEPS_CLJ_VERSION)
-            cp target/deps.clj-$VERSION-standalone.jar /tmp/release
-      - store_artifacts:
-          path: /tmp/release
-          destination: release
-  babashka:
-    docker:
-      # specify the version you desire here
-      - image: circleci/clojure:lein-2.8.1
-    working_directory: ~/repo
-    environment:
-      LEIN_ROOT: "true"
+      - when:
+          condition:
+            equal: [ <<parameters.uberjar-jdk>>, <<parameters.jdk>> ]
+          steps:
+            - run:
+                name: Create uberjar
+                command: |
+                  mkdir -p /tmp/release
+                  lein do clean, uberjar
+                  VERSION=$(cat resources/DEPS_CLJ_VERSION)
+                  cp target/deps.clj-$VERSION-standalone.jar /tmp/release
+            - store_artifacts:
+                path: /tmp/release
+                destination: release
+
+  mac-test:
+    # Run jvm and babsashka tests on the given JDK version.
+    parameters:
+      jdk:
+        type: string
+    macos:
+      xcode: 14.0.0
     steps:
       - checkout
-      - restore_cache:
-          keys:
-            - v1-dependencies-{{ checksum "deps.edn" }}
-            # fallback to using the latest cache if no exact match is found
-            - v1-dependencies-
+      - run:
+          name: Install <<parameters.jdk>>
+          command: |
+            brew install <<parameters.jdk>>
```

Comment:
Please don't use brew in CircleCI. This is very slow.

Reply:
(I did some quick survey of what else is available and put some brew package timings in the discussion further down below for consideration)

Reply:
Updated to use the `.circleci/script/install-clojure` script as suggested, installation time went down from 12 to 1 sec.